main.js

```diff
@@ -0,0 +1,141 @@
+/* eslint no-param-reassign: ["error", { "props": false }] */
+const boxes = document.querySelectorAll('.box');
+
+const Player = (name, symbol) => ({ name, symbol });
+const player1 = Player('Player 1', 'X');
+const player2 = Player('Player 2', 'O');
+
+const gameBoard = (() => {
+  let board = new Array(9).fill(0);
+
+  const disable = () => {
```

Comment:
More or less this is a logic that's better suited for the DOM. Please see my annotations below that implies why I suggest this.